addons/hr_homeworking/views/hr_employee_views.xml

```diff
@@ -85,12 +85,13 @@
                 </t>
                 <t t-set="location_map" t-value="{'home': 'fa-home', 'office': 'fa-building', 'other': 'fa-map-marker'}"/>
                 <t t-set="location" t-value="record.hr_icon_display.raw_value.split('_')[1]"/>
-                <t t-set="icon" t-value="location_map[location]"/>
                 <t t-if="location !== 'holiday'">
-                    <span class="fa fa-2x" t-attf-class="{{icon}} {{text_color}}" role="img"
+                    <t t-set="icon" t-value="location_map[location]"/>
+                    <span class="fa fa-2x" t-attf-class="{{icon}} {{text_color}} " role="img"
```

Comment:
What are these changes for?

Reply:
only set icon if location isn't 'holiday'

Reply:
> What are these changes for?

When the location is  a holiday, the icon is useless, so I have moved inside the if condition. is not good way ?

Reply:
Ah ok now I see what it's about. Sure it's a little useless, but it does no harm and changing it messes up the commit history and makes it less clear what this commit is actually about. Also by only changing it in the private view and not the public one, this two views are inconsistent.

I would prefer this change not to be there.

Reply:
> Ah ok now I see what it's about. Sure it's a little useless, but it does no harm and changing it messes up the commit history and makes it less clear what this commit is actually about. Also by only changing it in the private view and not the public one, this two views are inconsistent.
> 
> I would prefer this change not to be there.

Okay. i revert this changes thanks